bin/docker_start.sh

```diff
@@ -13,6 +13,9 @@ uwsgi_threads=${UWSGI_THREADS:-2}
 
 mountpoint=${SUBPATH:-/}
 
+# Make sure system checks are run
+python src/manage.py check
```

Comment:
@Floris272 by Sergei's suggestion I also added this to make sure the checks are run in docker